cognition_objects/message.py

```diff
@@ -189,3 +195,133 @@ def delete(project_id: str, message_id: str, with_commit: bool = True) -> None:
         CognitionMessage.id == message_id,
     ).delete()
     general.flush_or_commit(with_commit)
+
+
+def get_messages_per_conversation_query(project_id: str) -> str:
+    project_id = prevent_sql_injection(project_id, isinstance(project_id, str))
+    return f"""
+    SELECT 
+        conversation_id,
+        COUNT(*) messages,
+        c."header"
+    FROM cognition.message as m
+        inner join cognition.conversation c on c.id = m.conversation_id and c.project_id = m.project_id 
+    WHERE m.project_id = '{project_id}'
+    GROUP BY conversation_id, c."header" 
+    ORDER BY conversation_id, c."header"
+    """
+
+
+def get_response_time_messages_query(project_id: str) -> str:
+    project_id = prevent_sql_injection(project_id, isinstance(project_id, str))
+    return f"""
+    select round(sum(x) * 1000 / 500) * 500 / 1000 AS time_seconds, COUNT(*)
```

Comment:
sql keywords big (SELECT, FROM ,...)

There seem to be some calculatinos happening here that are unecessary every time so maybe
```suggestion
    -- round in ,5 steps
    select round(sum(x) * 2) /2  AS time_seconds, COUNT(*)
```